.github/workflows/main.yml

```diff
@@ -21,4 +21,17 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           slug: iExecBlockchainComputing/PoCo
-          
+      - name: Run static analysis with slither
+        uses: crytic/slither-action@v0.4.0
+        id: slither
+        with:
+          target: "contracts/tools/testing/slither/"
+          solc-version: '0.8.21'
```

Comment:
```suggestion
```

Reply:
Needed indeed : 
[-] SOLCVER was not set; guessing.
[-] Guessed 0.8.0.
Installing solc '0.8.0'...
Version '0.8.0' installed.

=> Error: Source file requires different compiler version (current compiler is 0.8.0+commit.c7dfd78e.Linux.g++) - note that nightly builds are considered to be strictly less than the released version
 --> node_modules/@openzeppelin/contracts-v5/interfaces/IERC5313.sol:4:1:
  |
4 | pragma solidity ^0.8.20;

Reply:
OK thanks :+1: